Block Entity/BEMixingBowl.cs

```diff
@@ -166,7 +166,16 @@ public override void Initialize(ICoreAPI api)
             {
                 CapacityLitres = Block.Attributes["capacityLitres"].AsInt(CapacityLitres);
             }
-
+            if (Block.Attributes["maxContainerSlotStackSize"].Exists == true)
+            {
+                MaxContainerSlotStackSize = Block.Attributes["maxContainerSlotStackSize"].AsInt(DefaultMaxContainerSlotStackSize);
+            }
+            else
+            {
+                // This is the default if no maxContainerSlotStackSize is specified.
+                // Needed for backwards compatibility.
```

Comment:
Simply removing the "if exists" portion of this block and making the above line this instead should work even in this case:

```csharp
Block.Attributes?["maxContainerSlotStackSize"]?.AsInt(DefaultMaxContainerSlotStackSize) ?? DefaultMaxContainerSlotStackSize;
```